kDriveCore/Data/Cache/AccountManager.swift

```diff
@@ -171,6 +171,7 @@ public class AccountManager: RefreshTokenDelegate, AccountManageable {
         }
     }
 
+    // TODO: Refactor init to explicit DriveContext
```

Comment:
Check with @PhilippeWeidmann about `sharedWithMe`

Reply:
DriveFileManager with sharedWithMe context is now correctly returned